packages/batcher/batcher-transaction-poster/src/mqtt/mqtt-publisher.ts

```diff
@@ -0,0 +1,55 @@
+// import { ENV } from '@paima/utils';
+import mqtt from 'mqtt';
+
+const wait = async (n: number): Promise<void> =>
+  await new Promise(resolve => setTimeout(resolve, n));
+
+export enum MQTTSystemEvents {
+  BATCHER_HASH = '/sys/batch_hash',
```

Comment:
https://docs.aws.amazon.com/whitepapers/latest/designing-mqtt-topics-aws-iot-core/mqtt-design-best-practices.html#:~:text=Since%20MQTT%20topics%20are%20case,when%20creating%20each%20topic%20level.

for paima events
/system/engine/stf
/system/batcher/hash/{hash}
...

for game events
/game/whatever